README.md

```diff
@@ -6,46 +6,192 @@
 
 ![](src/main/resources/META-INF/pluginIcon.svg)
 
+Git Machete plugin is a robust tool that simplify your git related workflow.
+It's a great completion of IntelliJ version control system.
+You can with ease rebase branches, do pull, push or reset.
+Just define what you would like your branch tree to look like and leave the rest to Git Machete.
```

Comment:
What `branch tree` is supposed to mean is yet unclear to the reader as of this point :/